shared/constants/swaps.js

```diff
@@ -114,6 +118,37 @@ export const SWAPS_CHAINID_CONTRACT_ADDRESS_MAP = {
   [RINKEBY_CHAIN_ID]: TESTNET_CONTRACT_ADDRESS,
 };
 
+export const SWAPS_WRAPPED_TOKENS_ADDRESSES = {
+  [MAINNET_CHAIN_ID]: WETH_CONTRACT_ADDRESS,
+  [SWAPS_TESTNET_CHAIN_ID]: WETH_CONTRACT_ADDRESS,
+  [BSC_CHAIN_ID]: WBNB_CONTRACT_ADDRESS,
+  [POLYGON_CHAIN_ID]: WMATIC_CONTRACT_ADDRESS,
+  [RINKEBY_CHAIN_ID]: WETH_CONTRACT_ADDRESS,
```

Comment:
Will the `WETH_CONTRACT_ADDRESS` work on rinkeby? WETH probably has a different address on rinkeby...

Reply:
yeah, this might be the token on rinkeby: https://rinkeby.etherscan.io/token/0xc778417e063141139fce010982780140aa0cd5ab